contracts/governance/InterchainGovernance.sol

```diff
@@ -28,17 +28,17 @@ contract InterchainGovernance is AxelarExecutable, TimeLock, Caller, IInterchain
 
     /**
      * @notice Initializes the contract
-     * @param gateway The address of the Axelar gateway contract
+     * @param gateway_ The address of the Axelar gateway contract
      * @param governanceChain_ The name of the governance chain
      * @param governanceAddress_ The address of the governance contract
      * @param minimumTimeDelay The minimum time delay for timelock operations
      */
     constructor(
-        address gateway,
+        address gateway_,
```

Comment:
what's the reason for changing this?

Reply:
because it inherits `Executable` which defined the `gateway` as public immutable